deny.toml

```diff
@@ -97,5 +97,5 @@ allow-git = [
     "https://github.com/alloy-rs/alloy",
     "https://github.com/paradigmxyz/evm-inspectors",
     "https://github.com/sigp/discv5",
-    "https://github.com/stevefan1999-personal/rust-igd",
+    "https://github.com/dariusc93/rust-igd",
```

Comment:
this can just be removed since it's not a git dep

Reply:
good catch